src/resources/js/event-editor.js

```diff
@@ -0,0 +1,24 @@
+var tribe_events_event_editor = tribe_events_event_editor || {};
+
+/**
+ * Implements behaviours that are specific to the event editor.
+ */
+( function( obj, $ ) {
+
+	var $sticky_in_month_view_checkbox = $( 'input[name="EventShowInCalendar"]' ),
+	    $featured_event_checkbox = $( 'input[name="feature_event"]' );
+
+	/**
+	 * If the 'feature event' box is checked, automatically check the
+	 * sticky-in-month-view box also.
+	 */
+	obj.auto_enable_sticky_field = function() {
+		if ( $( this ).prop( 'checked' ) ) {
```

Comment:
Return early if it isn't checked? I mean, we could, but in this case that would be more verbose as we're only interested in performing a single operation when the condition is met.